### ISSUE TITLE
Using className prop overrides defaults

### ISSUE THREAD

**Issue by dimon82** [helfi92/material-ui-treeview#6]
When I pass in `className` prop to both `expansionPanelSummaryProps` and `expansionPanelDetailsProps`, it overrides default styles and breaks the layout of the tree even when I set a simple `backgroundColor`.

It might be a desired behavior, but probably should be mentioned somewhere.

**Comment by helfi92** [comment]
It should technically only override whatever you are setting inside the class name.

_Example: Changing the background color_

```jsx
background: {
  "&:hover": {
    backgroundColor: 'aliceblue'
  },
  backgroundColor: 'aliceblue',
}

// ...

<MuiTreeView
  expansionPanelSummaryProps={{ className: classes.background }}
  listItemProps={{ className: classes.background }}
  tree={tree}
/>
```

If you are seeing something weird, let me know and I'll try to investigate.

**Comment by helfi92** [comment]
We should probably add a section in the `README` with examples (e.g., changing the background color) :)

**Comment by dimon82** [comment]
@helfi92 your example above overrides your default styles for `ExpansionPanelSummary`:

```
panelSummary: {
    padding: 0,
    marginLeft: theme.spacing.unit,
}
```

if I do something like `<MuiTreeView expansionPanelDetailsProps={{ className: classes.details }} ...` it'll override the styles below and all subnodes will be inline instead of block elements

```
panelDetails: {
    padding: 0,
    display: 'block',
}
```

**Comment by helfi92** [comment]
Ah, you're totally right. I will send a fix.

**Comment by helfi92** [comment]
Fixed in `v1.1.1`.

**Comment by helfi92** [comment]
@dimon82 Could you confirm it's working for you now?

**Comment by helfi92** [comment]
Closing as fixed in  `v1.1.1`. Please re-open if you are still experiencing the issue.
